dependencies.yaml

```diff
@@ -81,6 +87,13 @@ dependencies:
       - output_types: [conda, requirements]
         packages:
           - pre-commit
+  clang_tidy:
+    common:
+      - output_types: [conda, requirements]
+        packages:
+          - clang==15.0.7
+          - clang-tools==15.0.7
+          - make
```

Comment:
Can you use `ninja` instead? Tends to be faster.

Reply:
https://github.com/rapidsai/cuml/pull/5235/commits/26cbc00c23a6cb35d367f4e8e7a2bf78a99764fe